lib/sequin/config_parser.ex

```diff
@@ -9,20 +9,20 @@ defmodule Sequin.ConfigParser do
     |> put_redis_opts_ipv6(env)
   end
 
-  def log_level(env) do
+  @valid_log_levels [:error, :warning, :notice, :info, :debug]
+  def log_level(env, default_level) when default_level in @valid_log_levels do
     env
-    |> Map.get("LOG_LEVEL", "info")
+    |> Map.get("LOG_LEVEL", default_level |> to_string())
```

Comment:
Formatter issue - so sorry we didn't catch this! We should have a github action that runs on opened PRs now from outside contributors that will alert about this stuff (just need to rebase) 

```suggestion
    |> Map.get("LOG_LEVEL", to_string(default_level))
```